cmd/swarm/db.go

```diff
@@ -93,21 +93,6 @@ func dbImport(ctx *cli.Context) {
 	log.Info(fmt.Sprintf("successfully imported %d chunks", count))
 }
 
-func dbClean(ctx *cli.Context) {
```

Comment:
We really should be doing this automatically on start-up, when we detect the version/schema a user is running Swarm with.

If we detect a version of Swarm that we don't know, while we are in POC mode, we should just re-create the whole database from scratch.